.github/workflows/python.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.5, 3.6, 3.7, 3.8, 3.9, "pypy3"]
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
Let's also remove `Python 3.7` and mention the endoflife date https://devguide.python.org/versions/